README.md

```diff
@@ -61,3 +61,13 @@ runs:
         TESTS_TO_DISABLE: "flaky,known_issues"
       run: mvn test
 ```
+
+
+
+## Smoke test
+To run the smoke test, you can use the following command:
```

Comment:
Small change, could you please pluralise 'smoke test' to 'smoke tests' throughout this paragraph? As we plan to have more than one